server/render/index.js

```diff
@@ -82,6 +84,21 @@ export function render( element, key = JSON.stringify( element ) ) {
 		if ( ! renderedLayout ) {
 			bumpStat( 'calypso-ssr', 'loggedout-design-cache-miss' );
 			debug( 'cache miss for key', key );
+			if ( Math.random() < 0.001 || config.isEnabled( 'ssr/always-log-cache-misses' ) ) {
```

Comment:
I jumped through some hoops to convince myself that the `Math.random` comparison represents `1 log / 1000 requests`. Everything looks good there 👍 

It seems it would be nice to add config for this log sampling as well, `config.isEnabled( 'ssr/sample-log-cache-misses' ) && Math.random() < 0.001 || …`. What do you think?

Reply:
> It seems it would be nice to add config for this log sampling as well, config.isEnabled( 'ssr/sample-log-cache-misses' ) && Math.random() < 0.001 || …. What do you think?

:boom: Great idea. Then we can enable the logging in production in a separate, easy to revert PR.